### PR TITLE
ui: rebrand dev tools popups

### LINKED ISSUE #955: ️🛠 - Rebrand Dev tools popups
### Task description

No designs yet, but can be extrapolated from other popups

### Requirements

TBD

### LINKED ISSUE #993: Rebrand AliasConfirmation Popop

### LINKED ISSUE #1008: Rebrand FaucetRequest Popop

### LINKED ISSUE #1014: Rebrand MintNativeTokenConfirmation Popop

### LINKED ISSUE #1015: Rebrand MintNativeTokenForm Popop

### LINKED ISSUE #1016: Rebrand MintNftConfirmation Popop

### LINKED ISSUE #1017: Rebrand MintNftForm Popop

### LINKED ISSUE #1027: Rebrand TestDeepLinkForm Popop

### PR DESCRIPTION
**Pull request by MarkNerdi**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by Tuditi — APPROVED** [review]
LGTM! :blossom: :

File: packages/desktop/components/popup/popups/MintNftConfirmationPopup.svelte [hunk 1]
```diff
@@ -9,8 +9,9 @@
     import { formatTokenAmountPrecise } from '@core/token'
     import { buildNftOutputBuilderParams, mintNft, mintNftDetails } from '@core/wallet'
     import { PopupId, closePopup, openPopup } from '@desktop/auxiliary/popup'
-    import { Button, FontWeight, MediaPlaceholder, Tabs, Text, TextType } from '@ui'
+    import { MediaPlaceholder, Tabs } from '@ui'
```
Comment (nicole-obrien) [inline packages/desktop/components/popup/popups/MintNftConfirmationPopup.svelte:12]: Old Tabs component

File: packages/desktop/components/popup/popups/MintNftFormPopup.svelte [hunk 2]
```diff
@@ -13,8 +13,10 @@
     import { IMintNftDetails } from '@core/wallet'
     import { mintNftDetails, setMintNftDetails } from '@core/wallet/stores'
     import { PopupId, closePopup, openPopup } from '@desktop/auxiliary/popup'
-    import { Button, Error, FontWeight, OptionalInput, Text, TextInput, TextType, TooltipIcon } from '@ui'
+    import { Error, OptionalInput } from '@ui'
```
Comment (nicole-obrien) [inline packages/desktop/components/popup/popups/MintNftFormPopup.svelte:16]: Old optional input and error component

File: packages/desktop/components/popup/popups/MintNativeTokenFormPopup.svelte [hunk 3]
```diff
@@ -3,10 +3,12 @@
     import { localize } from '@core/i18n'
     import { setMintTokenDetails, mintTokenDetails, IMintTokenDetails } from '@core/wallet'
     import { closePopup, openPopup, PopupId } from '@desktop/auxiliary/popup'
-    import { Button, Error, NumberInput, Text, TextInput, OptionalInput, FontWeight, AliasInput, TextType } from '@ui'
+    import { Error, OptionalInput, AliasInput } from '@ui'
```
Comment (nicole-obrien) [inline packages/desktop/components/popup/popups/MintNativeTokenFormPopup.svelte:6]: Old error and optional inpit

File: packages/desktop/components/popup/popups/FaucetRequestPopup.svelte [hunk 4]
```diff
@@ -1,14 +1,17 @@
 <script lang="ts">
     import { requestTokensFromFaucet } from '@contexts/developer'
     import { localize } from '@core/i18n'
-    import { nodeInfo } from '@core/network'
+    import { network } from '@core/network'
     import { closePopup } from '@desktop/auxiliary/popup'
-    import { Button, Error, Text, FontWeight, TextType } from '@ui'
+    import { Error } from '@ui'
```
Comment (nicole-obrien) [inline packages/desktop/components/popup/popups/FaucetRequestPopup.svelte:6]: Old error

**Review by nicole-obrien — CHANGES_REQUESTED** [review]

**Comment by MarkNerdi** [comment]
@nicole-obrien Your comments are fixed in https://github.com/bloomwalletio/bloom/pull/977

**Pull request merged**